app/flows/check_travel_during_coronavirus_flow/outcomes/results.erb

```diff
@@ -16,6 +16,15 @@
     } %>
   </div>
 
+  <div class="govuk-grid-row travel-abroad__section">
+    <div class="govuk-grid-column-one-third">&nbsp;</div>
```

Comment:
Looks like you need an empty column here. This is a reasonable way to do it except that on mobile this column will be one character tall, which might not be ideal.

An alternative is to add a custom class to set a very small height only on desktop, something like this.

```
<div class="govuk-grid-column-one-third desktop-min-height"></div>

.desktop-min-height {
  @include govuk-media-query($from: tablet) {
      min-height: 1px;
   }
}
```

The `$from` might need to be `desktop`, you'd have to check when it collapses (I think it collapses on tablet though).

Reply:
Thanks for the example code! I've tried implementing it with both `tablet` and `desktop` and they both look the same. The mobile view stacks correctly for both though.